cfpb/consumerfinance.gov
# Add rental assistance finder tool

Imports external rental assistance finder tool and makes it available in Wagtail.

## Additions

- RAF tool Wagtail block and template
- RAF tool JS
- Gulp task to copy RAF tool JS to on-demand directory  


## How to test this PR

1. Check out branch and run migrations
2. Create browse page and add RAF Tool block
3. Verify that tool shows on the page and works as expected


## Screenshots

### Rental assistance finder tool in a Wagtail browse page

<img width="1129" alt="Screen Shot 2021-07-26 at 9 51 52 PM" src="https://user-images.githubusercontent.com/778171/127097468-a140b426-ad9d-4928-8ce1-1ccd27b9e144.png">


## Notes and todos

- A future iteration will import the JS in an npm module, add the CSS as a file rather than a style block in the tool block's template, and pull the JSON data from s3 rather than bundling it in the tool's main js file.


## Checklist


### Front-end testing


#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
